qualtran/bloqs/rotations/phase_gradient.py

```diff
@@ -138,6 +169,19 @@ def decompose_from_registers(
         )
 
 
+@bloq_example
+def _phase_gradient_state() -> PhaseGradientState:
+    from qualtran import QFxp
```

Comment:
hmm pylint doesn't like this since it's redundant with the module-level import. I think just add a pylint ignore comment here for now until we have a better answer to these sorts of issues

https://docs.pylint.org/faq.html#message-control

Reply:
Ok cool, I added a disable right above the offending function. It did not feel right to just put it at the top without any reference to the function that was causing the pylint warning. If it should be at the top with a comment explaining the offending function let me know and I will move it.